src/_data/toc/compliance.yml

```diff
@@ -0,0 +1,20 @@
+label: Compliance
+pages:
+    - label: Industry Compliance
+      url: ../../../compliance/industry-compliance.html
```

Comment:
```suggestion
      url: /compliance/industry-compliance.html
```

Reply:
Fixed.

Reply:
@ccondra, you removed the forward slash at the beginning of the file path, which will cause link errors. You must start the file path with a forward slash for each TOC entry.